### ISSUE TITLE
Don't crop on movies with different perspectives?

### ISSUE THREAD

**Issue by vr8hub** [lisamelton/video_transcoding#258]
I think I know the answer to this, but just making sure.

For movies that vary their perspective, e.g. Dark Knight, I assume we shouldn't crop at all when transcoding. Correct?

(I did a few searches here and couldn't find anything; I did find a reference to Dark Knight and Grand Budhapest Hotel in an old issue where --no-constrain-crop was made the default, but it didn't reference specifically what to do with them.)

**Comment by donmelton** [comment]
@vr8hub Correct, more or less. :) For "The Dark Knight (2008)" I would recommend not cropping at all. Same for "The Hunger Games: Catching Fire (2013)". But for "The Grand Budapest Hotel (2014)" I would recommend a crop of "`24:22:2:2`" because there still is some unused area around the edges.

**Comment by vr8hub** [comment]
I don't actually have GPH (I have the other two), but I'm curious–if it has different perspectives, how are you determining the recommended crop?

**Comment by donmelton** [comment]
@vr8hub Sorry I took so long to reply! Had an appointment with a contractor. Anyway...

I'm using `detect-crop` and then tinkering with the result. Because... I'm a control freak. :)

And the key to proper use of `detect-crop` are those `mpv` preview commands it spews out. I don't commit to any crop without trying those.

In the case of TDK, the `HandBrakeCLI` result from `detect-crop` is way too aggressive, so I go with the `ffmpeg` result, which is "`0:0:0:0`".

In the case of TGBH, the `HandBrakeCLI` result from `detect-crop` is _again_ way too aggressive, so I start with the `ffmpeg` result, which is "`22:22:2:2`". But to my eye, trimming a little more off the top, like a good Army haircut, looks better. :)
